service/metrics/config.go

```diff
@@ -15,25 +15,45 @@ type Config struct {
 	Collect map[string]Collector
 }
 
+type NamedCollector struct {
+	// Name of the collector
+	Name string `json:"name"`
+
+	// Collector structure
+	Collector `json:"collector"`
```

Comment:
Not sure I remember how JSON annotation works on an inherited struct, does it work at or all or it's redundant?

Reply:
yeah, It's not redundant. Under `collector` would be inherited structure with json tagged fields